.github/workflows/ks-review.yml

```diff
@@ -189,6 +190,7 @@ jobs:
           SERVICE_PORT: ${{ matrix.port }}
           K8S_SECRET_ALLOWED_HOSTS: "*"
           APP_MIGRATE_COMMAND: ${{ matrix.database == true && '/app/.prod/on_deploy.sh' || ''}}
+          LOGOUT_REDIRECT_URL: ${{ env.EMPLOYER_URL }}/login?logout=true
```

Comment:
This one is still missing from in bf-review and te-review. te-review its `ADMIN_URL`